include/votca/csg/beadtriple.h

```diff
@@ -0,0 +1,65 @@
+/* 
+ * Copyright 2009-2011 The VOTCA Development Team (http://www.votca.org)
```

Comment:
Sorry, I copied beadpair.h and edited it. There the copyright was 2009-2011 as well